v2.1/start-a-node.md

```diff
@@ -62,6 +62,7 @@ Flag | Description
 `--advertise-addr` | The IP address/hostname and port to tell other nodes to use. If using a hostname, it must be resolvable from all nodes. If using an IP address, it must be routable from all nodes; for IPv6, use the notation `[...]`, e.g., `[::1]` or `[fe80::f6f2:::]`.<br><br>This flag's effect depends on how it is used in combination with `--listen-addr`. For example, if the port number is different than the one used in `--listen-addr`, port forwarding is required. For more details, see [Networking](recommended-production-settings.html#networking).<br><br>**Default:** The value of `--listen-addr`; if `--listen-addr` is not specified, advertises the node's canonical hostname and port `26257`
 `--listen-addr` | The IP address/hostname and port to listen on for connections from other nodes and clients. For IPv6, use the notation `[...]`, e.g., `[::1]` or `[fe80::f6f2:::]`.<br><br>This flag's effect depends on how it is used in combination with `--advertise-addr`. For example, the node will also advertise itself to other nodes using this value if `--advertise-addr` is not specified. For more details, see [Networking](recommended-production-settings.html#networking).<br><br>**Default:** Listen on all IP addresses on port `26257`; if `--advertise-addr` is not specified, also advertise the node's canonical hostname to other nodes
 `--http-addr` | The IP address/hostname and port to listen on for Admin UI HTTP requests. For IPv6, use the notation `[...]`, e.g., `[::1]:8080` or `[fe80::f6f2:::]:8080`.<br><br>**Default:** Listen on the address part of `--listen-addr` on port `8080`
+`--locality-advertise-addr` | <span class="version-tag">New in v2.1:</span> The IP address/hostname and port to tell other nodes in specific localities to use. This flag is useful when running a cluster across multiple networks, where nodes in a given network have access to a private or local interface while nodes outside the network do not. In this case, you can can use `--locality-advertise-addr` to tell nodes within the same network to prefer the private or local address to improve performance and use `--advertise-addr` to tell nodes outside the network to use another address that is reachable from them.<br><br>This feature relies on nodes being started with the [`--locality`](#locality) flag. The flag should be specified as a commma-separated list of `locality@address` values, for example:<br><br>`--locality-advertise-addr=region=us-west@127.0.0.1:26257,datacenter=us-west-1b@127.0.0.1:26258`<br><br>See the [example](#start-a-multi-node-cluster-across-private-networks) below for more details.
```

Comment:
127.0.0.1 is not a realistic example here. I'd use different addresses in 10.0.0.0/8:

`--locality-advertise-addr=region=us-west@10.1.2.3:26257,region=us-east@10.5.6.7:26257`

(While comma-separated lists are theoretically possible here, I'm not sure if they'd ever be used in practice. In the overwhelming majority of cases, a node will have one entry in `locality-advertise-addr` for one of its own locality fields)

Reply:
Thanks, @bdarnell. Fixed. Though I used `10.0.0.0:26257` in the example, not `10.0.0.0/8:26257`. That ok? Wasn't clear on that.

Reply:
Right, I didn't mean for you to use `/8`. `10.0.0.0/8` describes the private network in use; addresses in that network don't have the `/8`.